### PR TITLE
(PC-5626) visuel cas passant

### PR DESCRIPTION
**Pull request by Giregm**
Link to JIRA ticket: https://passculture.atlassian.net/browse/PC-5626

## Checklist

I have:

- [x] Made sure the title of my PR follows the convention `[PC-XXX] <summary>`.
- [x] Made sure my feature is working on the relevant real / virtual devices.
- [x] Written **unit tests** for my feature.
- [x] Added new reusable components to the AppComponents page and communicate to the team on slack
- [x] Added a **screenshot** for UI tickets.

## Screenshots

| \*iOS - [iPhone5s] | \*iOS  | android | Little - [Phone name] |
| -------------------- | :----------------------: | --------------------: | --------------------: |
 |   image| image |image  |                       |

## Deploy hard

If native code (ios/android) was modified, **after** the PR is merged, on the master branch, upgrade the **app version** (+1 patch):

- if you want an hard deployment of the testing environment, use `yarn version:testing` (this will create a commit with a tag)
- then run `git push --follow-tags`

### CODE REVIEW

File: src/features/offer/components/OfferHero.tsx [hunk 1]
```diff
@@ -13,21 +15,47 @@ interface Props {
...
-        extraHeight={top}
-        blurRadius={Platform.OS === 'android' ? 5 : 20}
-        resizeMode={'cover'}
-        source={{ uri: imageUrl }}
-      />
-      {/** Add 1 pixel to avoid 1 white pixel on androids */}
-      <Rectangle size={screenWidth + 1} />
-    </React.Fragment>
+    <HeroContainer>
+      <React.Fragment>
```
Comment (antoinewg) [inline src/features/offer/components/OfferHero.tsx:19]: plus besoin du fragment

File: src/features/offer/components/OfferHero.tsx [hunk 2]
```diff
@@ -3,7 +3,9 @@ import { Dimensions, Platform } from 'react-native'
 import styled from 'styled-components/native'
 
 import { Rectangle } from 'ui/svg/Rectangle'
-import { getSpacing } from 'ui/theme'
+import { getShadow } from 'ui/theme'
+import { ColorsEnum, getSpacing, Spacer } from 'ui/theme'
```
Comment (antoinewg) [inline src/features/offer/components/OfferHero.tsx:7]: même endroit  que getShadow

**Review by antoinewg — APPROVED** [review]
2 petites remarques

File: src/features/offer/components/OfferHero.tsx (hunk 1 above)
Comment (Giregm) [reply src/features/offer/components/OfferHero.tsx:19]: bien vu !

**Pull request merged**